app/controllers/groups/samples/file_imports_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Groups
+  module Samples
+    # Controller actions for Group Samples File Import Controller
+    class FileImportsController < Groups::ApplicationController
+      include SampleFileImportActions
```

Comment:
Remove the `if Flipper.enabled?(:batch_sample_file_import)` from `config/routes/group.rb` and only include SampleFileImportActions `if Flipper.enabled?(:batch_sample_file_import)`